_docs/migrate_module.md

```diff
@@ -10,11 +10,11 @@ You will have someone by your side in this process. The general flow is to…
 * Ask one of the Administrators to add you to the `incoming-migrations` team.
 * At that point you can transfer your own repository.
 * If migrating a module from puppetlabs, re-enable github issues.
-* Verify that all webhooks except travis are disabled.
+* Verify that all webhooks are disabled.
 * Update the README.md with a description of the deprecation and a link to the new module location.
 * Release a copy of your module to the 'puppet' forge account.
-* Add the module to our [modulesync setup](managed_modules).
-* Add the module to our [plumbing repository](plumbing)(handles travis secrets).
+* Add the module to our [modulesync setup](publish_setup).
+* Add the module to our [plumbing repository](plumbing).
```

Comment:
Since we've removed Travis secrets in plumbing, I think we should drop this altogether and only rely on modulesync.